### ISSUE TITLE
OpenBLAS 0.3.5: recipe for target 'kernel' failed

### ISSUE THREAD

**Issue by eugeneswalker** [LLNL/spack#22342]
`openblas@0.3.5` fails to build using:
* `spack@develop` (d7f1ff68fba047380b7fc81319c369eebf6b1b81 from `Mon Mar 15 16:41:24 2021 -0500`)
* Ubuntu 18.04, x86_64, GCC 7.5.0
* RHEL 7, x86_64, GCC 8.1.0
* RHEL 8, x86_64, GCC 8.3.1

Concrete spec: [openblas-dbnxbc.spec.yaml.txt](https://github.com/spack/spack/files/6152307/openblas-dbnxbc.spec.yaml.txt)
Build log: [openblas-spack-build-out.txt](https://github.com/spack/spack/files/6152310/openblas-spack-build-out.txt)

```
$> spack install -f ./openblas-dbnxbc.spec.yaml
...
==> Installing openblas-0.3.5-dbnxbcy2iprdlrcucelvklx33uf6vyd2
==> Fetching https://spack-llnl-mirror.s3-us-west-2.amazonaws.com/_source-cache/archive/09/0950c14bd77c90a6427e26210d6dab422271bc86f9fc69126725833ecdaa0e85.tar.gz
################################################################################################################################################ 100.0%
==> Applied patch /opt/spack/var/spack/repos/builtin/packages/openblas/0001-use-usr-bin-env-perl.patch
==> openblas: Executing phase: 'edit'
==> openblas: Executing phase: 'build'
==> Error: ProcessError: Command exited with status 2:
    'make' '-j16' 'CC=/opt/spack/lib/spack/env/gcc/gcc' 'FC=/opt/spack/lib/spack/env/gcc/gfortran' 'MAKE_NB_JOBS=0' 'ARCH=x86_64' 'DYNAMIC_ARCH=1' 'DYNAMIC_OLDER=1' 'TARGET=GENERIC' 'USE_OPENMP=0' 'USE_THREAD=0' 'libs' 'netlib' 'shared'

3 errors found in build log:
     1895    /opt/spack/lib/spack/env/gcc/gcc -O2 -DMAX_STACK_ALLOC=2048 -Wall -m64 -DF_INTERFACE_GFORT -fPIC -DDYNAMIC_ARCH -DDYNAMIC_OLDER -DNO_WARM
             UP -DMAX_CPU_NUMBER=192 -DMAX_PARALLEL_NUMBER=1 -DVERSION=\"0.3.5\" -DASMNAME=zgeadd_k_PRESCOTT -DASMFNAME=zgeadd_k_PRESCOTT_ -DNAME=zgea
             dd_k_PRESCOTT_ -DCNAME=zgeadd_k_PRESCOTT -DCHAR_NAME=\"zgeadd_k_PRESCOTT_\" -DCHAR_CNAME=\"zgeadd_k_PRESCOTT\" -DNO_AFFINITY -DTS=_PRESCO
             TT -I.. -DBUILD_KERNEL -DTABLE_NAME=gotoblas_PRESCOTT -DDOUBLE  -DCOMPLEX -c -DDOUBLE -DCOMPLEX -UROWM ../kernel/x86_64/../generic/zgeadd
             .c -o zgeadd_k_PRESCOTT.o
     1896    /opt/spack/lib/spack/env/gcc/gcc -c -O2 -DMAX_STACK_ALLOC=2048 -Wall -m64 -DF_INTERFACE_GFORT -fPIC -DDYNAMIC_ARCH -DDYNAMIC_OLDER -DNO_W
             ARMUP -DMAX_CPU_NUMBER=192 -DMAX_PARALLEL_NUMBER=1 -DVERSION=\"0.3.5\" -DASMNAME=zneg_tcopy_PRESCOTT -DASMFNAME=zneg_tcopy_PRESCOTT_ -DNA
             ME=zneg_tcopy_PRESCOTT_ -DCNAME=zneg_tcopy_PRESCOTT -DCHAR_NAME=\"zneg_tcopy_PRESCOTT_\" -DCHAR_CNAME=\"zneg_tcopy_PRESCOTT\" -DNO_AFFINI
             TY -DTS=_PRESCOTT -I.. -DBUILD_KERNEL -DTABLE_NAME=gotoblas_PRESCOTT -DDOUBLE  -DCOMPLEX generic/zneg_tcopy_2.c -o zneg_tcopy_PRESCOTT.o
     1897    /opt/spack/lib/spack/env/gcc/gcc -c -O2 -DMAX_STACK_ALLOC=2048 -Wall -m64 -DF_INTERFACE_GFORT -fPIC -DDYNAMIC_ARCH -DDYNAMIC_OLDER -DNO_W
             ARMUP -DMAX_CPU_NUMBER=192 -DMAX_PARALLEL_NUMBER=1 -DVERSION=\"0.3.5\" -DASMNAME=zlaswp_ncopy_PRESCOTT -DASMFNAME=zlaswp_ncopy_PRESCOTT_
             -DNAME=zlaswp_ncopy_PRESCOTT_ -DCNAME=zlaswp_ncopy_PRESCOTT -DCHAR_NAME=\"zlaswp_ncopy_PRESCOTT_\" -DCHAR_CNAME=\"zlaswp_ncopy_PRESCOTT\"
              -DNO_AFFINITY -DTS=_PRESCOTT -I.. -DBUILD_KERNEL -DTABLE_NAME=gotoblas_PRESCOTT -DDOUBLE  -DCOMPLEX generic/zlaswp_ncopy_2.c -o zlaswp_n
             copy_PRESCOTT.o
     1898    /opt/spack/lib/spack/env/gcc/gcc -c -O2 -DMAX_STACK_ALLOC=2048 -Wall -m64 -DF_INTERFACE_GFORT -fPIC -DDYNAMIC_ARCH -DDYNAMIC_OLDER -DNO_W
             ARMUP -DMAX_CPU_NUMBER=192 -DMAX_PARALLEL_NUMBER=1 -DVERSION=\"0.3.5\" -DASMNAME=setparam_PRESCOTT -DASMFNAME=setparam_PRESCOTT_ -DNAME=s
             etparam_PRESCOTT_ -DCNAME=setparam_PRESCOTT -DCHAR_NAME=\"setparam_PRESCOTT_\" -DCHAR_CNAME=\"setparam_PRESCOTT\" -DNO_AFFINITY -DTS=_PRE
             SCOTT -I.. -DBUILD_KERNEL -DTABLE_NAME=gotoblas_PRESCOTT -UDOUBLE  -UCOMPLEX -DUSE_GEMM3M setparam_PRESCOTT.c -o setparam_PRESCOTT.o
     1899    ar  -ru ../libopenblas-r0.3.5.a samax_k_PRESCOTT.o samin_k_PRESCOTT.o smax_k_PRESCOTT.o smin_k_PRESCOTT.o isamax_k_PRESCOTT.o isamin_k_PR
             ESCOTT.o ismax_k_PRESCOTT.o ismin_k_PRESCOTT.o sasum_k_PRESCOTT.o saxpy_k_PRESCOTT.o scopy_k_PRESCOTT.o sdot_k_PRESCOTT.o sdsdot_k_PRESCO
             TT.o dsdot_k_PRESCOTT.o snrm2_k_PRESCOTT.o srot_k_PRESCOTT.o sscal_k_PRESCOTT.o sswap_k_PRESCOTT.o saxpby_k_PRESCOTT.o sgemv_n_PRESCOTT.o
              sgemv_t_PRESCOTT.o ssymv_U_PRESCOTT.o ssymv_L_PRESCOTT.o sger_k_PRESCOTT.o sgemm_kernel_PRESCOTT.o sgemm_incopy_PRESCOTT.o sgemm_itcopy_
             PRESCOTT.o sgemm_oncopy_PRESCOTT.o sgemm_otcopy_PRESCOTT.o sgemm_beta_PRESCOTT.o strmm_kernel_LN_PRESCOTT.o strmm_kernel_LT_PRESCOTT.o st
             rmm_kernel_RN_PRESCOTT.o strmm_kernel_RT_PRESCOTT.o strsm_kernel_LN_PRESCOTT.o strsm_kernel_LT_PRESCOTT.o strsm_kernel_RN_PRESCOTT.o strs
             m_kernel_RT_PRESCOTT.o strmm_iunucopy_PRESCOTT.o strmm_iunncopy_PRESCOTT.o strmm_ilnucopy_PRESCOTT.o strmm_ilnncopy_PRESCOTT.o strmm_iutu
             copy_PRESCOTT.o strmm_iutncopy_PRESCOTT.o strmm_iltucopy_PRESCOTT.o strmm_iltncopy_PRESCOTT.o strmm_ounucopy_PRESCOTT.o strmm_ounncopy_PR
             ESCOTT.o strmm_olnucopy_PRESCOTT.o strmm_olnncopy_PRESCOTT.o strmm_outucopy_PRESCOTT.o strmm_outncopy_PRESCOTT.o strmm_oltucopy_PRESCOTT.
             o strmm_oltncopy_PRESCOTT.o strsm_iunucopy_PRESCOTT.o strsm_iunncopy_PRESCOTT.o strsm_ilnucopy_PRESCOTT.o strsm_ilnncopy_PRESCOTT.o strsm
             _iutucopy_PRESCOTT.o strsm_iutncopy_PRESCOTT.o strsm_iltucopy_PRESCOTT.o strsm_iltncopy_PRESCOTT.o strsm_ounucopy_PRESCOTT.o strsm_ounnco
             py_PRESCOTT.o strsm_olnucopy_PRESCOTT.o strsm_olnncopy_PRESCOTT.o strsm_outucopy_PRESCOTT.o strsm_outncopy_PRESCOTT.o strsm_oltucopy_PRES
             COTT.o strsm_oltncopy_PRESCOTT.o ssymm_iutcopy_PRESCOTT.o ssymm_iltcopy_PRESCOTT.o ssymm_outcopy_PRESCOTT.o ssymm_oltcopy_PRESCOTT.o soma
             tcopy_k_cn_PRESCOTT.o somatcopy_k_rn_PRESCOTT.o somatcopy_k_ct_PRESCOTT.o somatcopy_k_rt_PRESCOTT.o simatcopy_k_cn_PRESCOTT.o simatcopy_k
             _rn_PRESCOTT.o simatcopy_k_ct_PRESCOTT.o simatcopy_k_rt_PRESCOTT.o sgeadd_k_PRESCOTT.o sneg_tcopy_PRESCOTT.o slaswp_ncopy_PRESCOTT.o setp
             aram_PRESCOTT.o damax_k_PRESCOTT.o damin_k_PRESCOTT.o dmax_k_PRESCOTT.o dmin_k_PRESCOTT.o idamax_k_PRESCOTT.o idamin_k_PRESCOTT.o idmax_k
             _PRESCOTT.o idmin_k_PRESCOTT.o dasum_k_PRESCOTT.o daxpy_k_PRESCOTT.o dcopy_k_PRESCOTT.o ddot_k_PRESCOTT.o dnrm2_k_PRESCOTT.o drot_k_PRESC
             OTT.o dscal_k_PRESCOTT.o dswap_k_PRESCOTT.o daxpby_k_PRESCOTT.o dgemv_n_PRESCOTT.o dgemv_t_PRESCOTT.o dsymv_U_PRESCOTT.o dsymv_L_PRESCOTT
             .o dger_k_PRESCOTT.o dgemm_kernel_PRESCOTT.o dgemm_oncopy_PRESCOTT.o dgemm_otcopy_PRESCOTT.o dgemm_beta_PRESCOTT.o dtrmm_kernel_LN_PRESCO
             TT.o dtrmm_kernel_LT_PRESCOTT.o dtrmm_kernel_RN_PRESCOTT.o dtrmm_kernel_RT_PRESCOTT.o dtrsm_kernel_LN_PRESCOTT.o dtrsm_kernel_LT_PRESCOTT
             .o dtrsm_kernel_RN_PRESCOTT.o dtrsm_kernel_RT_PRESCOTT.o dtrmm_iunucopy_PRESCOTT.o dtrmm_iunncopy_PRESCOTT.o dtrmm_ilnucopy_PRESCOTT.o dt
             rmm_ilnncopy_PRESCOTT.o dtrmm_iutucopy_PRESCOTT.o dtrmm_iutncopy_PRESCOTT.o dtrmm_iltucopy_PRESCOTT.o dtrmm_iltncopy_PRESCOTT.o dtrmm_oun
             ucopy_PRESCOTT.o dtrmm_ounncopy_PRESCOTT.o dtrmm_olnucopy_PRESCOTT.o dtrmm_olnncopy_PRESCOTT.o dtrmm_outucopy_PRESCOTT.o dtrmm_outncopy_P
             RESCOTT.o dtrmm_oltucopy_PRESCOTT.o dtrmm_oltncopy_PRESCOTT.o dtrsm_iunucopy_PRESCOTT.o dtrsm_iunncopy_PRESCOTT.o dtrsm_ilnucopy_PRESCOTT
             .o dtrsm_ilnncopy_PRESCOTT.o dtrsm_iutucopy_PRESCOTT.o dtrsm_iutncopy_PRESCOTT.o dtrsm_iltucopy_PRESCOTT.o dtrsm_iltncopy_PRESCOTT.o dtrs
             m_ounucopy_PRESCOTT.o dtrsm_ounncopy_PRESCOTT.o dtrsm_olnucopy_PRESCOTT.o dtrsm_olnncopy_PRESCOTT.o dtrsm_outucopy_PRESCOTT.o dtrsm_outnc
             opy_PRESCOTT.o dtrsm_oltucopy_PRESCOTT.o dtrsm_oltncopy_PRESCOTT.o dsymm_iutcopy_PRESCOTT.o dsymm_iltcopy_PRESCOTT.o dsymm_outcopy_PRESCO
             TT.o dsymm_oltcopy_PRESCOTT.o domatcopy_k_cn_PRESCOTT.o domatcopy_k_rn_PRESCOTT.o domatcopy_k_ct_PRESCOTT.o domatcopy_k_rt_PRESCOTT.o dim
             atcopy_k_cn_PRESCOTT.o dimatcopy_k_rn_PRESCOTT.o dimatcopy_k_ct_PRESCOTT.o dimatcopy_k_rt_PRESCOTT.o dgeadd_k_PRESCOTT.o dneg_tcopy_PRESC
             OTT.o dlaswp_ncopy_PRESCOTT.o camax_k_PRESCOTT.o camin_k_PRESCOTT.o icamax_k_PRESCOTT.o icamin_k_PRESCOTT.o casum_k_PRESCOTT.o caxpy_k_PR
             ESCOTT.o caxpyc_k_PRESCOTT.o ccopy_k_PRESCOTT.o cdotc_k_PRESCOTT.o cdotu_k_PRESCOTT.o cnrm2_k_PRESCOTT.o csrot_k_PRESCOTT.o cscal_k_PRESC
             OTT.o cswap_k_PRESCOTT.o caxpby_k_PRESCOTT.o cgemv_n_PRESCOTT.o cgemv_t_PRESCOTT.o cgemv_r_PRESCOTT.o cgemv_c_PRESCOTT.o cgemv_o_PRESCOTT
             .o cgemv_u_PRESCOTT.o cgemv_s_PRESCOTT.o cgemv_d_PRESCOTT.o csymv_U_PRESCOTT.o csymv_L_PRESCOTT.o chemv_U_PRESCOTT.o chemv_L_PRESCOTT.o c
             hemv_V_PRESCOTT.o chemv_M_PRESCOTT.o cgeru_k_PRESCOTT.o cgerc_k_PRESCOTT.o cgerv_k_PRESCOTT.o cgerd_k_PRESCOTT.o cgemm_kernel_n_PRESCOTT.
             o cgemm_kernel_r_PRESCOTT.o cgemm_kernel_l_PRESCOTT.o cgemm_kernel_b_PRESCOTT.o cgemm_incopy_PRESCOTT.o cgemm_itcopy_PRESCOTT.o cgemm_onc
             opy_PRESCOTT.o cgemm_otcopy_PRESCOTT.o cgemm_beta_PRESCOTT.o ctrmm_kernel_LN_PRESCOTT.o ctrmm_kernel_LT_PRESCOTT.o ctrmm_kernel_LR_PRESCO
             TT.o ctrmm_kernel_LC_PRESCOTT.o ctrmm_kernel_RN_PRESCOTT.o ctrmm_kernel_RT_PRESCOTT.o ctrmm_kernel_RR_PRESCOTT.o ctrmm_kernel_RC_PRESCOTT
             .o ctrsm_kernel_LN_PRESCOTT.o ctrsm_kernel_LT_PRESCOTT.o ctrsm_kernel_LR_PRESCOTT.o ctrsm_kernel_LC_PRESCOTT.o ctrsm_kernel_RN_PRESCOTT.o
              ctrsm_kernel_RT_PRESCOTT.o ctrsm_kernel_RR_PRESCOTT.o ctrsm_kernel_RC_PRESCOTT.o cgemm3m_kernel_PRESCOTT.o ctrmm_iunucopy_PRESCOTT.o ctr
             mm_iunncopy_PRESCOTT.o ctrmm_ilnucopy_PRESCOTT.o ctrmm_ilnncopy_PRESCOTT.o ctrmm_iutucopy_PRESCOTT.o ctrmm_iutncopy_PRESCOTT.o ctrmm_iltu
             copy_PRESCOTT.o ctrmm_iltncopy_PRESCOTT.o ctrmm_ounucopy_PRESCOTT.o ctrmm_ounncopy_PRESCOTT.o ctrmm_olnucopy_PRESCOTT.o ctrmm_olnncopy_PR
             ESCOTT.o ctrmm_outucopy_PRESCOTT.o ctrmm_outncopy_PRESCOTT.o ctrmm_oltucopy_PRESCOTT.o ctrmm_oltncopy_PRESCOTT.o ctrsm_iunucopy_PRESCOTT.
             o ctrsm_iunncopy_PRESCOTT.o ctrsm_ilnucopy_PRESCOTT.o ctrsm_ilnncopy_PRESCOTT.o ctrsm_iutucopy_PRESCOTT.o ctrsm_iutncopy_PRESCOTT.o ctrsm
             _iltucopy_PRESCOTT.o ctrsm_iltncopy_PRESCOTT.o ctrsm_ounucopy_PRESCOTT.o ctrsm_ounncopy_PRESCOTT.o ctrsm_olnucopy_PRESCOTT.o ctrsm_olnnco
             py_PRESCOTT.o ctrsm_outucopy_PRESCOTT.o ctrsm_outncopy_PRESCOTT.o ctrsm_oltucopy_PRESCOTT.o ctrsm_oltncopy_PRESCOTT.o csymm_iutcopy_PRESC
             OTT.o csymm_iltcopy_PRESCOTT.o csymm_outcopy_PRESCOTT.o csymm_oltcopy_PRESCOTT.o chemm_iutcopy_PRESCOTT.o chemm_iltcopy_PRESCOTT.o chemm_
             outcopy_PRESCOTT.o chemm_oltcopy_PRESCOTT.o cgemm3m_incopyb_PRESCOTT.o cgemm3m_itcopyb_PRESCOTT.o cgemm3m_incopyr_PRESCOTT.o cgemm3m_itco
             pyr_PRESCOTT.o cgemm3m_incopyi_PRESCOTT.o cgemm3m_itcopyi_PRESCOTT.o cgemm3m_oncopyb_PRESCOTT.o cgemm3m_otcopyb_PRESCOTT.o cgemm3m_oncopy
             r_PRESCOTT.o cgemm3m_otcopyr_PRESCOTT.o cgemm3m_oncopyi_PRESCOTT.o cgemm3m_otcopyi_PRESCOTT.o csymm3m_iucopyb_PRESCOTT.o csymm3m_oucopyb_
             PRESCOTT.o csymm3m_iucopyr_PRESCOTT.o csymm3m_oucopyr_PRESCOTT.o csymm3m_iucopyi_PRESCOTT.o csymm3m_oucopyi_PRESCOTT.o csymm3m_ilcopyb_PR
             ESCOTT.o csymm3m_olcopyb_PRESCOTT.o csymm3m_ilcopyr_PRESCOTT.o csymm3m_olcopyr_PRESCOTT.o csymm3m_ilcopyi_PRESCOTT.o csymm3m_olcopyi_PRES
             COTT.o chemm3m_iucopyb_PRESCOTT.o chemm3m_oucopyb_PRESCOTT.o chemm3m_iucopyr_PRESCOTT.o chemm3m_oucopyr_PRESCOTT.o chemm3m_iucopyi_PRESCO
             TT.o chemm3m_oucopyi_PRESCOTT.o chemm3m_ilcopyb_PRESCOTT.o chemm3m_olcopyb_PRESCOTT.o chemm3m_ilcopyr_PRESCOTT.o chemm3m_olcopyr_PRESCOTT
             .o chemm3m_ilcopyi_PRESCOTT.o chemm3m_olcopyi_PRESCOTT.o comatcopy_k_cn_PRESCOTT.o comatcopy_k_rn_PRESCOTT.o comatcopy_k_ct_PRESCOTT.o co
             matcopy_k_rt_PRESCOTT.o comatcopy_k_cnc_PRESCOTT.o comatcopy_k_rnc_PRESCOTT.o comatcopy_k_ctc_PRESCOTT.o comatcopy_k_rtc_PRESCOTT.o cimat
             copy_k_cn_PRESCOTT.o cimatcopy_k_rn_PRESCOTT.o cimatcopy_k_ct_PRESCOTT.o cimatcopy_k_rt_PRESCOTT.o cimatcopy_k_cnc_PRESCOTT.o cimatcopy_k
             _rnc_PRESCOTT.o cimatcopy_k_ctc_PRESCOTT.o cimatcopy_k_rtc_PRESCOTT.o cgeadd_k_PRESCOTT.o cneg_tcopy_PRESCOTT.o claswp_ncopy_PRESCOTT.o z
             amax_k_PRESCOTT.o zamin_k_PRESCOTT.o izamax_k_PRESCOTT.o izamin_k_PRESCOTT.o zasum_k_PRESCOTT.o zaxpy_k_PRESCOTT.o zaxpyc_k_PRESCOTT.o zc
             opy_k_PRESCOTT.o zdotc_k_PRESCOTT.o zdotu_k_PRESCOTT.o znrm2_k_PRESCOTT.o zdrot_k_PRESCOTT.o zscal_k_PRESCOTT.o zswap_k_PRESCOTT.o zaxpby
             _k_PRESCOTT.o zgemv_n_PRESCOTT.o zgemv_t_PRESCOTT.o zgemv_r_PRESCOTT.o zgemv_c_PRESCOTT.o zgemv_o_PRESCOTT.o zgemv_u_PRESCOTT.o zgemv_s_P
             RESCOTT.o zgemv_d_PRESCOTT.o zsymv_U_PRESCOTT.o zsymv_L_PRESCOTT.o zhemv_U_PRESCOTT.o zhemv_L_PRESCOTT.o zhemv_V_PRESCOTT.o zhemv_M_PRESC
             OTT.o zgeru_k_PRESCOTT.o zgerc_k_PRESCOTT.o zgerv_k_PRESCOTT.o zgerd_k_PRESCOTT.o zgemm_kernel_n_PRESCOTT.o zgemm_kernel_r_PRESCOTT.o zge
             mm_kernel_l_PRESCOTT.o zgemm_kernel_b_PRESCOTT.o zgemm_oncopy_PRESCOTT.o zgemm_otcopy_PRESCOTT.o zgemm_beta_PRESCOTT.o ztrmm_kernel_LN_PR
             ESCOTT.o ztrmm_kernel_LT_PRESCOTT.o ztrmm_kernel_LR_PRESCOTT.o ztrmm_kernel_LC_PRESCOTT.o ztrmm_kernel_RN_PRESCOTT.o ztrmm_kernel_RT_PRES
             COTT.o ztrmm_kernel_RR_PRESCOTT.o ztrmm_kernel_RC_PRESCOTT.o ztrsm_kernel_LN_PRESCOTT.o ztrsm_kernel_LT_PRESCOTT.o ztrsm_kernel_LR_PRESCO
             TT.o ztrsm_kernel_LC_PRESCOTT.o ztrsm_kernel_RN_PRESCOTT.o ztrsm_kernel_RT_PRESCOTT.o ztrsm_kernel_RR_PRESCOTT.o ztrsm_kernel_RC_PRESCOTT
             .o zgemm3m_kernel_PRESCOTT.o ztrmm_iunucopy_PRESCOTT.o ztrmm_iunncopy_PRESCOTT.o ztrmm_ilnucopy_PRESCOTT.o ztrmm_ilnncopy_PRESCOTT.o ztrm
             m_iutucopy_PRESCOTT.o ztrmm_iutncopy_PRESCOTT.o ztrmm_iltucopy_PRESCOTT.o ztrmm_iltncopy_PRESCOTT.o ztrmm_ounucopy_PRESCOTT.o ztrmm_ounnc
             opy_PRESCOTT.o ztrmm_olnucopy_PRESCOTT.o ztrmm_olnncopy_PRESCOTT.o ztrmm_outucopy_PRESCOTT.o ztrmm_outncopy_PRESCOTT.o ztrmm_oltucopy_PRE
             SCOTT.o ztrmm_oltncopy_PRESCOTT.o ztrsm_iunucopy_PRESCOTT.o ztrsm_iunncopy_PRESCOTT.o ztrsm_ilnucopy_PRESCOTT.o ztrsm_ilnncopy_PRESCOTT.o
              ztrsm_iutucopy_PRESCOTT.o ztrsm_iutncopy_PRESCOTT.o ztrsm_iltucopy_PRESCOTT.o ztrsm_iltncopy_PRESCOTT.o ztrsm_ounucopy_PRESCOTT.o ztrsm_
             ounncopy_PRESCOTT.o ztrsm_olnucopy_PRESCOTT.o ztrsm_olnncopy_PRESCOTT.o ztrsm_outucopy_PRESCOTT.o ztrsm_outncopy_PRESCOTT.o ztrsm_oltucop
             y_PRESCOTT.o ztrsm_oltncopy_PRESCOTT.o zsymm_iutcopy_PRESCOTT.o zsymm_iltcopy_PRESCOTT.o zsymm_outcopy_PRESCOTT.o zsymm_oltcopy_PRESCOTT.
             o zhemm_iutcopy_PRESCOTT.o zhemm_iltcopy_PRESCOTT.o zhemm_outcopy_PRESCOTT.o zhemm_oltcopy_PRESCOTT.o zgemm3m_incopyb_PRESCOTT.o zgemm3m_
             itcopyb_PRESCOTT.o zgemm3m_incopyr_PRESCOTT.o zgemm3m_itcopyr_PRESCOTT.o zgemm3m_incopyi_PRESCOTT.o zgemm3m_itcopyi_PRESCOTT.o zgemm3m_on
             copyb_PRESCOTT.o zgemm3m_otcopyb_PRESCOTT.o zgemm3m_oncopyr_PRESCOTT.o zgemm3m_otcopyr_PRESCOTT.o zgemm3m_oncopyi_PRESCOTT.o zgemm3m_otco
             pyi_PRESCOTT.o zsymm3m_iucopyb_PRESCOTT.o zsymm3m_oucopyb_PRESCOTT.o zsymm3m_iucopyr_PRESCOTT.o zsymm3m_oucopyr_PRESCOTT.o zsymm3m_iucopy
             i_PRESCOTT.o zsymm3m_oucopyi_PRESCOTT.o zsymm3m_ilcopyb_PRESCOTT.o zsymm3m_olcopyb_PRESCOTT.o zsymm3m_ilcopyr_PRESCOTT.o zsymm3m_olcopyr_
             PRESCOTT.o zsymm3m_ilcopyi_PRESCOTT.o zsymm3m_olcopyi_PRESCOTT.o zhemm3m_iucopyb_PRESCOTT.o zhemm3m_oucopyb_PRESCOTT.o zhemm3m_iucopyr_PR
             ESCOTT.o zhemm3m_oucopyr_PRESCOTT.o zhemm3m_iucopyi_PRESCOTT.o zhemm3m_oucopyi_PRESCOTT.o zhemm3m_ilcopyb_PRESCOTT.o zhemm3m_olcopyb_PRES
             COTT.o zhemm3m_ilcopyr_PRESCOTT.o zhemm3m_olcopyr_PRESCOTT.o zhemm3m_ilcopyi_PRESCOTT.o zhemm3m_olcopyi_PRESCOTT.o zomatcopy_k_cn_PRESCOT
             T.o zomatcopy_k_rn_PRESCOTT.o zomatcopy_k_ct_PRESCOTT.o zomatcopy_k_rt_PRESCOTT.o zomatcopy_k_cnc_PRESCOTT.o zomatcopy_k_rnc_PRESCOTT.o z
             omatcopy_k_ctc_PRESCOTT.o zomatcopy_k_rtc_PRESCOTT.o zimatcopy_k_cn_PRESCOTT.o zimatcopy_k_rn_PRESCOTT.o zimatcopy_k_ct_PRESCOTT.o zimatc
             opy_k_rt_PRESCOTT.o zimatcopy_k_cnc_PRESCOTT.o zimatcopy_k_rnc_PRESCOTT.o zimatcopy_k_ctc_PRESCOTT.o zimatcopy_k_rtc_PRESCOTT.o zgeadd_k_
             PRESCOTT.o zneg_tcopy_PRESCOTT.o zlaswp_ncopy_PRESCOTT.o
     1900    ar: `u' modifier ignored since `D' is the default (see `U')
  >> 1901    ar: strmm_kernel_LN_PRESCOTT.o: No such file or directory
     1902    ../Makefile.tail:52: recipe for target 'kernel' failed
  >> 1903    make[1]: *** [kernel] Error 1
     1904    make[1]: Leaving directory '/tmp/root/spack-stage/spack-stage-openblas-0.3.5-dbnxbcy2iprdlrcucelvklx33uf6vyd2/spack-src/kernel'
     1905    Makefile:145: recipe for target 'libs' failed
  >> 1906    make: *** [libs] Error 1

See build log for details:
  /tmp/root/spack-stage/spack-stage-openblas-0.3.5-dbnxbcy2iprdlrcucelvklx33uf6vyd2/spack-build-out.txt
```

`openblas@0.3.5` builds fine before 5b2a54952c083dbb1f6ea45827adbd83d023fbbb but not after.

@sethrj ?

**Comment by sethrj** [comment]
Ughh I’ll look into this.

**Comment by sethrj** [comment]
@eugeneswalker When I updated the recipe I only tested 0.3.12 and 0.3.13 . This looks like a bug with the older openblas version -- I reproduced it on my machine, though I'm baffled by the error itself since that object file is clearly being built. I even tried building with a single job just in case it's a timing issue but no luck. I'll just amend my change so that it only applies to 0.3.12 and newer.
